packaging/flatpak/org.openwinecomponents.umu.launcher.yml

```diff
@@ -143,14 +148,14 @@ modules:
   - name: umu-run
     buildsystem: simple
     build-commands:
-      - install -D umu-run-cli /app/bin/umu-run
-      - install -D umu-launcher.tar.gz /app/share/umu/umu-launcher.tar.gz
+      - |
+        git submodule update --init --recursive
+        ./configure.sh --prefix=/app
+        make install
```

Comment:
The required `FLATPAK=xtrue` control variable to disable manpages is missing here. For reference: https://github.com/Open-Wine-Components/umu-launcher/pull/80/files#diff-04d351e49f21347db94177694691013189c0c7e751a2894b523df166b3b5b5c8R154